boromir674/music-album-creator
# Music Album Creator v1.4.1 Release

1.4.1 (2023-04-15)
==================

Changes
-------

test
""""
- use 'Lavf59.27.100' as the expected encoder ffmpeg will use when segmenting a webm file
- retry 3 times before raising Time Out Error in test_downloading

ci
""
- install ffmpeg on macos
- pass List Job
- fix tox
- enable automated pypi pulbish
